src/main/io/vtx_control.c

```diff
@@ -47,6 +47,7 @@ PG_RESET_TEMPLATE(vtxConfig_t, vtxConfig,
       .smartAudioEarlyAkkWorkaroundEnable = SETTING_VTX_SMARTAUDIO_EARLY_AKK_WORKAROUND_DEFAULT,
       .smartAudioAltSoftSerialMethod = SETTING_VTX_SMARTAUDIO_ALTERNATE_SOFTSERIAL_METHOD_DEFAULT,
       .softSerialShortStop = SETTING_VTX_SOFTSERIAL_SHORTSTOP_DEFAULT,
+      .smartAudioStopBits = SETTING_VTX_SMARTAUDIO_STOPBITS_DEFAULT,
```

Comment:
@erstec sorry, forgot we also need to bump vtxConfig PG version from 3 to 4. See line 43

Reply:
@DzikuVx updated